index.html

```diff
@@ -2489,6 +2489,16 @@ <h3 property="schema:name">Social Web Working Group</h3>
         </p>
       </section>
 
+      <section id="security-recursive-objects">
+        <h2>Recursive Objects</h2>
+        <p>
+	  Servers should take care to properly handle recursive ActivityStreams
```

Comment:
I think it's better as it's a bit more concrete on the problem and how to handle it 👍